activesupport/lib/active_support/core_ext/date/conversions.rb

```diff
@@ -27,7 +27,6 @@ class Date
   #   date = Date.new(2007, 11, 10)       # => Sat, 10 Nov 2007
   #
   #   date.to_formatted_s(:db)            # => "2007-11-10"
-  #   date.to_formatted_s(:db)            # => "2007-11-10"
```

Comment:
Can you use `date.to_fs(:db)` instead?

Reply:
Sure @kamipo I'll add thanks.

Reply:
@kamipo I made the requested changes - https://github.com/rails/rails/pull/43798/commits/0055b72a3b83dddd2a5477719f479b3071509524